### PR TITLE
Enable multipath for NVMfPath.

### PR DESCRIPTION
**Pull request by artek-koltun**
Enables Multipath after API transition in https://github.com/opiproject/opi-spdk-bridge/pull/461

Example
```
# Create controller
docker run --network=host --rm -it namely/grpc-cli call --json_input --json_output $BRIDGE_ADDR CreateNVMfRemoteController "{nv_mf_remote_controller : {multipath: 'NVME_MULTIPATH_MULTIPATH'}, nv_mf_remote_controller_id: 'nvmetcp12'}"

# Create paths
docker run --network=host --rm -it namely/grpc-cli call --json_input --json_output $BRIDGE_ADDR CreateNVMfPath "{nv_mf_path : {controller_id: {value: '//storage.opiproject.org/volumes/nvmetcp12'}, traddr:'$STORAGE_TARGET_IP', subnqn:'nqn.2019-07.io.spdk:target0', trsvcid:'4444', trtype:'NVME_TRANSPORT_TCP', adrfam:'NVMF_ADRFAM_IPV4', hostnqn:'nqn.2014-08.org.nvmexpress:uuid:feb98abe-d51f-40c8-b348-2753f3571d3c'}, nv_mf_path_id: 'nvmetcp12path0'}"
docker run --network=host --rm -it namely/grpc-cli call --json_input --json_output $BRIDGE_ADDR CreateNVMfPath "{nv_mf_path : {controller_id: {value: '//storage.opiproject.org/volumes/nvmetcp12'}, traddr:'$STORAGE_TARGET_IP', subnqn:'nqn.2019-07.io.spdk:target0', trsvcid:'4445', trtype:'NVME_TRANSPORT_TCP', adrfam:'NVMF_ADRFAM_IPV4', hostnqn:'nqn.2014-08.org.nvmexpress:uuid:feb98abe-d51f-40c8-b348-2753f3571d3c'}, nv_mf_path_id: 'nvmetcp12path1'}"

# check SPDK
./rpc.py bdev_nvme_get_controllers -n nvmetcp12
[
  {
    "name": "nvmetcp12",
    "ctrlrs": [
      {
        "state": "enabled",
        "trid": {
          "trtype": "TCP",
          "adrfam": "IPv4",
          "traddr": "10.91.52.142",
          "trsvcid": "4444",
          "subnqn": "nqn.2019-07.io.spdk:target0"
        },
        "cntlid": 12,
        "host": {
          "nqn": "nqn.2014-08.org.nvmexpress:uuid:feb98abe-d51f-40c8-b348-2753f3571d3c",
          "addr": "",
          "svcid": ""
        }
      },
      {
        "state": "enabled",
        "trid": {
          "trtype": "TCP",
          "adrfam": "IPv4",
          "traddr": "10.91.52.142",
          "trsvcid": "4445",
          "subnqn": "nqn.2019-07.io.spdk:target0"
        },
        "cntlid": 13,
        "host": {
          "nqn": "nqn.2014-08.org.nvmexpress:uuid:feb98abe-d51f-40c8-b348-2753f3571d3c",
          "addr": "",
          "svcid": ""
        }
      }
    ]
  }
]

```

### CODE REVIEW

File: pkg/backend/nvme_path.go [hunk 1]
```diff
@@ -57,6 +57,16 @@ func (s *Server) CreateNVMfPath(_ context.Context, in *pb.CreateNVMfPathRequest)
...
 
 	multipath := ""
+	if numberOfPaths, ok := s.Volumes.NvmeNumberOfPaths[in.NvMfPath.ControllerId.Value]; ok {
+		if numberOfPaths < 1 {
+			log.Printf("error: Entry with number of paths for %v exists with zero paths", in.NvMfPath.ControllerId.Value)
+			return nil, status.Error(codes.Internal, "Internal error")
+		}
+		// set multipath parameter only when at least one path already exists
+		multipath = strings.ToLower(
+			strings.ReplaceAll(controller.Multipath.String(), "NVME_MULTIPATH_", ""),
```
Comment (glimchb) [inline pkg/backend/nvme_path.go:67]: usually you make the `replace` code a function, no ?

Comment (artek-koltun) [reply pkg/backend/nvme_path.go:67]: only if it is used in more than one place

Comment (artek-koltun) [reply pkg/backend/nvme_path.go:67]: but I don't mind extracting a func for this

Comment (artek-koltun) [reply pkg/backend/nvme_path.go]: Updated

**Review by glimchb — APPROVED** [review]
lgtm

**Pull request merged**